polaris-react/src/components/AppProvider/global.css

```diff
@@ -110,11 +110,9 @@ html {
   system font and then define font-families and rem-based
   font-sizes on descendant elements:
 */
-@supports (font: -apple-system-body) {
-  @media (--p-breakpoints-sm-down) {
-    html {
-      font: -apple-system-body;
-    }
+@supports (font: -apple-system-body) and (-webkit-touch-callout: none) {
```

Comment:
Presumably this would also target iPad devices?

Some of the bigger iPads can go up to 2732x2048, which is quite a lot higher than the `sm` breakpoint of 490px (even accounting for @2x resolution scale) - is it intended to also use `font: -apple-system-body` on those larger devices?

Reply:
Really good point @jesstelford 

@davebcn87 do you have any thoughts here? I would guess "no", we don't want to be using `font: -apple-system-body` on iPad sizes that large?

Perhaps we should cap with a set screen width (`md`)?

Reply:
> do different devices (iPhone/iPad) or screen resolutions make adjustment for the system text styles (caption, body, etc)?

A good question that I'm not sure about. The effects of `font: -apple-system-body` seem to be poorly documented unfortunately.

Might be safe to assume it works the same across all devices, and if we're happy this change will effect iPads, then let's :shipit: